css/css-images/image-orientation/image-orientation-none-cross-origin-canvas.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Images Module Level 3: image-orientation: none</title>
+<script src=/common/get-host-info.sub.js></script>
+<link rel="author" title="Noam Rosenthal" href="mailto:noam@webkit.org">
+<link rel="help" href="https://github.com/w3c/csswg-drafts/issues/5165">
+<link rel="match" href="reference/image-orientation-none-cross-origin-ref.html">
+<meta name=fuzzy content="10;100">
+<style>
+    img {display: none}
+    canvas {
+        width: 100px;
+        height: 100px;
+    }
+</style>
+<script>
+    const src1 = 'support/exif-orientation-1-ul.jpg';
```

Comment:
These images aren't already present in the repo, nor are they part of this PR; did you accidentally leave them out of the commit?

Reply:
They are already present in the repo.
See https://github.com/web-platform-tests/wpt/tree/master/css/css-images/image-orientation/support

Reply:
Ahhh, I was looking at wpt.fyi, but that appears to only reflect folders that actually contain a test. Sorry about that.